holoviews/core/options.py

```diff
@@ -224,13 +236,6 @@ def _base_opts(self, *args, **kwargs):
 
         # By default do not clone in .opts method
         clone = kwargs.get('clone', None)
-
-        if apply_groups and config.future_deprecations:
-            msg = ("Calling the .opts method with options broken down by options "
-                   "group (i.e. separate plot, style and norm groups) is deprecated. "
-                   "Use the .options method converting to the simplified format "
-                   "instead or use hv.opts.apply_groups for backward compatibility.")
-            param.main.warning(msg)
```

Comment:
Is there now any future warning for the deprecated formats? I know there was the issue with the recursive calls which meant it was always triggered when enabled but I had hoped there was a way around this.

Reply:
Nevermind, same confusion as the previous comment about ``__call__``. I can't visualize who calls what anymore, I'd really need a diagram to see how the different formats get converted and then pencil in the warnings at the appropriate stages.